### ISSUE TITLE
Add form for Company profile creation

### ISSUE THREAD

**Issue by swetshaw** [Developer-DAO/DAO-job-board#24]
This form is required to onboard the companies who want to post job openings on Developer DAO.
A few points that are in my mind right now are:

- Name of company
- Brief detail
- Year of founding
- Industry (finance, games, healthcare, NFT, etc)
- Company Size
- Website
- Twitter
- LinkedIn

**Comment by angeljgomezc** [comment]
YES! The list of data you added is exactly what we need for each company's page. 

If you want to take this issue, let me know.

Otherwise, I'll start working on it tomorrow and drop a design here before coding :)

**Comment by swetshaw** [comment]
I am already working on creating company profile page #19 . So you can take up this issue. 😃
